pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -54,6 +54,10 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 					Verbs:     util.NewStringSet("get", "list", "watch"),
 					Resources: util.NewStringSet(authorizationapi.PolicyOwnerGroupName, authorizationapi.KubeAllGroupName, authorizationapi.OpenshiftStatusGroupName, authorizationapi.KubeStatusGroupName),
 				},
+				{
+					Verbs:     util.NewStringSet("get"),
+					Resources: util.NewStringSet("pods/exec"),
```

Comment:
It'd be good to define a group for pods/exec here: https://github.com/deads2k/origin/blob/add-update-attribute-striction/pkg/authorization/api/types.go#L53 so that we at least know it exists by looking at that file.